### DISCUSSION TITLE
Track group titles should be sticky

### DISCUSSION THREAD

**Discussion by wirednkod** [jeluard/swipealot#631] [Ideas]
Upon scrolling down on the provided tracks, the titles (as seen in the screen below, should be sticky; This way user can easily choose one of those select-box options for each list, without the need to scroll back up (same function like "All tracks" select box).
<img width="1703" alt="Screenshot 2023-08-05 at 20 45 55" src="https://github.com/paritytech/governance-ui/assets/5408605/60cc38ad-7f95-42db-9b66-bb837a1b7c3e">

**Comment by rossbulat** [comment]
➕ 1
This would complement the suggestions in https://github.com/paritytech/governance-ui/discussions/591 very well.
